src/stylesheets/components/_pane.scss

```diff
@@ -9,15 +9,6 @@
     }
   }
 
-  .app-pane__nav {
-    @include govuk-media-query($from: tablet) {
-      display: flex;
```

Comment:
The flex stuff was no longer used, so the only thing this was doing was setting the background colour.